virttest/libvirt_xml/accessors.py

```diff
@@ -786,7 +786,12 @@ def __call__(self, value):
             # Allow other classes to generate parent structure
             parent = self.xmltreefile().find(self.parent_xpath)
             if parent is None:
-                raise xcepts.LibvirtXMLNotFoundError
+                # Create parent xpath if not exists
+                self.xmltreefile().create_by_xpath(self.parent_xpath)
```

Comment:
It is a little hard to find a suitable test case to verify this change. 
Considering this is a common enhancement, and similar workaround has already been tested in vm_xml.py like below, I evaluate this change has low risk. So I approve it and would like to observe if any other cases will fail against this.
                     ```
   if numa_number is not None and numa_number > 0:
                            vmcpu_xml.xmltreefile.create_by_xpath('/numa')
                            vmcpu_xml.numa_cell = vmcpu_xml.dicts_to_cells(nodexml_list)
```